File: Submissions/T07/M1.cpp
#include <stdio.h>

int main() {
    int nilai[5];

    printf("Masukkan 5 nilai:\n");

    for(int i = 0; i < 5; i++) {
        scanf("%d", &nilai[i]);
    }

    float avg = 0;

    for(int i = 0; i < 5; i++) {
        avg += nilai[i];
    }

    avg /= 5;

    if(avg >= 80) printf("Rata-rata: %.2f\nGrade: A", avg);
    else if(avg >= 70) printf("Rata-rata: %.2f\nGrade: B", avg);
    else if(avg >= 60) printf("Rata-rata: %.2f\nGrade: C", avg);
    else if(avg >= 50) printf("Rata-rata: %.2f\nGrade: D", avg);
    else printf("Rata-rata: %.2f\nGrade: E", avg);

    return 0;
}

File: Submissions/T07/M1.py
nilai = list(map(int, input("Masukkan 5 nilai : ").split()))

rata_rata = sum(nilai) / 5

if rata_rata >= 80:
    grade = "A"
elif rata_rata >= 70:
    grade = "B"
elif rata_rata >= 60:
    grade = "C"
elif rata_rata >= 50:
    grade = "D"
else:
    grade = "E"

print(f"Rata-rata : {rata_rata:.2f}")
print("Grade :", grade)